website/docs/d/policy_definition.html.markdown

```diff
@@ -24,19 +24,29 @@ output "id" {
 
 ## Argument Reference
 
-* `display_name` - Specifies the name of the Policy Definition.
+* `name` - Specifies the name of the Policy Definition. Conflicts with `display_name`.
+
+* `display_name` - Specifies the display name of the Policy Definition. Conflicts with `name`.
+
+~> **NOTE** Since `display_name` does not have uniqueness on Azure, therefore when using `display_name` to retrieve existing policy set definitions, only one of the policy set definitions with same `display_name` will be retrieved. 
```

Comment:
This should be a caution that and error will happen once we start erroring out

Reply:
Done